src/components/chat/Chat.tsx

```diff
@@ -0,0 +1,18 @@
+import ChatHeader from './ChatHeader';
+import ChatContent from './ChatContent';
+import ChatInput from './ChatInput';
+import { useParams } from "react-router-dom";
+function Main(): JSX.Element {

```

Comment:
```suggestion
const Main = ():JSX.Element => {
```

보통 함수형 코드를 짤 때 function으로 선언하기 보다는, 화살표 함수를 많이 씁니다!!
일반함수를 쓸때보다 화살표 함수를 쓰면 일반함수를 쓸 때보다 예측 가능한 프로그래밍을 할 수 있기 때문인데요,
관련해서 `화살표 함수와 일반 함수의 차이` 를 알아보시면 좋을 것 같아요!